src/Augurk.UI/_Imports.razor

```diff
@@ -7,4 +7,8 @@
 @using Microsoft.AspNetCore.Components.WebAssembly.Http
 @using Microsoft.JSInterop
 @using Augurk.UI
+@using Augurk.UI.Components
```

Comment:
Missing license

Reply:
Unfortunately it seems that the `file_header_template` we've added to the `.editorconfig` only applies to C# and VB files, but not to Razor, CSS, or other files. So we'll have to do these manually, or remove them entirely I guess. @marktaling Thoughts?

Reply:
Half manual / half automatic doesn't sound all to great to me. There are some vscode extensions out there which also do this for other file types, perhaps we should take a look at those?

Reply:
I've added an extension to the workspace recommendations and configured it for Razor and CSS files.